server/config/sessions/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  MONGO: require('./mongo'),
```

Comment:
My bad - I derp-ed big time. 

It doesn't require ES6 syntax! I didn't `require` the correct object. 😑 

Garh. I pushed the changes.